packages/runtime-core/__tests__/helpers/useTemplateRef.spec.ts

```diff
@@ -70,18 +70,20 @@ describe('useTemplateRef', () => {
     expect(t1!.value).toBe(null)
   })
 
-  test('should warn on duplicate useTemplateRef', () => {
+  test('should warn and return same value on duplicate useTemplateRef', () => {
+    let f1, f2
     const root = nodeOps.createElement('div')
     render(
       h(() => {
-        useTemplateRef('foo')
-        useTemplateRef('foo')
+        f1 = useTemplateRef('foo')
+        f2 = useTemplateRef('foo')
         return ''
```

Comment:
I think it should be `return h('div', { ref: 'foo' })` instead of return an empty string.

Reply:
I've made tweaks see https://github.com/vuejs/core/pull/12883/commits/6ca14f9c47b8e91bb6b321b7e66c9d5c6728525f